bin/main.rb

```diff
@@ -0,0 +1,5 @@
+# A file which has a shebang line as its first line is
+# granted execute permission.
```

Comment:
- [ ] The shebangline is: `#!/usr/bin/env ruby`, this "text" was an explanation, you can delete these comments.